partner_event/__init__.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
+
```

Comment:
It's OK to remove the coding comment, but then :fire: all whitespace at the start of the file please.